modules/ref-dashboard-configuration-options.adoc

```diff
@@ -21,6 +36,14 @@ disableCustomServingRuntimes` | `false` | Shows the *Settings → Serving runtim
 | `dashboardConfig:
 disableDistributedWorkloads` | `false` | Shows the *Distributed workloads* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
 | `dashboardConfig:
+disableHardwareProfiles` (Technology Preview) | `true` | Hides the *Settings → Hardware profiles* option in the dashboard navigation menu, and shows the *Settings → Accelerator profiles* option if that option is enabled. 
+
+To show the *Settings → Hardware profiles* option in the dashboard navigation men, set the `disableHardwareProfiles` value to `false`.
+
+If both options are enabled, the `disableHardwareProfiles` option overrides the `disableAcceleratorProfiles` option.
```

Comment:
suggest 
"If both options are enabled..."
change to
"If both options are set to `false`..."
?
if I'm interpreting the logic correctly

Reply:
Updated, thank you!